applications/luci-app-babeld/luasrc/model/cbi/babeld.lua

```diff
@@ -0,0 +1,126 @@
+m = Map("babeld", translate("Babeld Overview"))
+m.pageaction = false
+
+s = m:section(NamedSection, "__babeldoverview__")
+
+function s.render(self, sid)
+	local tpl = require "luci.template"
+
+	tpl.render_string([[
+		<%
+		local utl = require "luci.util"
+		local v6_port = 33123
+		local dump = utl.exec("(echo dump | nc ::1 %d)" % v6_port)
```

Comment:
You still owe me an answer. Is the `echo` necessary?

Reply:
Yes. If I remove it, the output is not correct? Or is there another way? I copied that from luci-app-olsr ;)

Reply:
So I tested it. And it did not work.

Reply:
I think you need to redirect stdout or stderr.
I think the echo is not necessary here.
But as I said, I can't try it out.
You can write the difference in a file.

But that should not bother use. It's just an extra fork that I think is not neccessary.
Maybe we should write a FIXME comment above it.
If someone has the time and the inclination, they can take another look at it.

Reply:
Added fixme comment. I will rewrite that when we have IPC via ubus merged. :)